config/jobs/kubernetes-sigs/slack-infra/slack-infra-config.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-sigs/slack-infra:
+  - name: pull-slack-infra-verify
+    decorate: true
+    path_alias: sigs.k8s.io/slack-infra
+    always_run: true
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
```

Comment:
Can we use a less horrible image? What exactly from this image do you need? Seems like you could just as well just use a golang image